scripts/supported_versions.py

```diff
@@ -5,9 +5,10 @@
 """
 
 _es2x = "2.4.4"
-_es5x = "5.5.2"
+_es5x = "5.5.2"  # TODO: shall we update this constant to 5.6.13 ?
```

Comment:
yes, please

Reply:
The thing is it will probably complicate our tests against released files because they use the version in file name... I will look at this.

Reply:
> The thing is it will probably complicate our tests against released files because they use the version in file name... I will look at this.

then don't worry about it